src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -126,7 +129,12 @@ typedef int         (*t8_forest_vtk_cell_data_kernel) (t8_forest_t forest,
                                                        modus);
 
 void
```

Comment:
This function should return something. Similar to the old vtk write function.

Writing to a file can always go wrong and the return value of this function should reflect this.
I.e. return 1 if successful, 0 if not.
The write () routine of the vtk xml writer will give you the corresponding output.
Could this also go wrong before this call? If so, we need to return 0 then as well (and clean up the memory).

Reply:
I am not sure what else we should check. The other vtk functions do not return any information about whether the task was successful or not.

Reply:
Ok, then we should stick to write() and return 1 if it was successful and 0 otherwise